### PR TITLE
SW-2110 Planting site search capability

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

File: src/components/PlantingSites/PlantingSitesList.tsx [hunk 1]
```diff
@@ -36,15 +39,29 @@ export default function PlantingSitesList(props: PlantingSitesListProps): JSX.El
...
+            type: 'Fuzzy',
+            values: [debouncedSearchTerm],
+          },
+        ],
       },
       count: 0,
     };
     const apiSearchResults = await search(params);
+    if (!debouncedSearchTerm) {
+      setPlantingSites(apiSearchResults);
```
Comment (charlie-tf) [inline src/components/PlantingSites/PlantingSitesList.tsx:61]: If I'm understanding correctly, is `plantingSites` the full list of planting sites when no search term is supplied?

Comment (constanzauanini) [reply src/components/PlantingSites/PlantingSitesList.tsx:61]: Yes, we need to save the full list to show or not the empty state, and in other place save the results when search term is applied, to show results in table

File: src/components/PlantingSites/PlantingSitesList.tsx [hunk 2]
```diff
@@ -36,15 +39,29 @@ export default function PlantingSitesList(props: PlantingSitesListProps): JSX.El
...
+        operation: 'and',
+        children: [
+          {
+            field: 'organization_id',
+            operation: 'field',
+            values: [organization.id],
+          },
+          {
+            operation: 'field',
+            field: 'name',
```
Comment (karthikbtf) [inline src/components/PlantingSites/PlantingSitesList.tsx:51]: suggestion for query:
OR group for name or description

**Review by karthikbtf — APPROVED** [review]
Thank you!!!

**Pull request merged**